tests/functional_tests/ffront_tests/test_execution.py

```diff
@@ -479,6 +479,28 @@ def scalar_arg(scalar_inp: float64) -> Field[[Vertex], float64]:
     assert np.allclose(ref, out.array())
 
 
+def test_nested_scalar_arg(fieldview_backend):
```

Comment:
Would it make sense to add another test for operators with multiple inputs mixing scalar and field arguments?

Reply:
Given how the lowering works and balancing between number of tests and coverage I would say it's not necessary. Your call.

Reply:
Ok, makes sense.